SingleCellPatch/extract_patches.py

```diff
@@ -133,102 +136,25 @@ def generate_mask(positions, positions_labels, cell_id, window, window_segmentat
         target_mask.reshape((x_size, y_size, 1)), \
         target_mask2.reshape((x_size, y_size, 1))
 
-
-def instance_clustering(cell_segmentation, 
```

Comment:
I saw that you moved this function to the pipeline script. I guess to make it cleaner I can just isolate this function to a separate script so we can leave everything in the pipeline folder as simple helper functions

Reply:
Now moved to script `SingleCellPatch/instance_clustering.py`

Reply:
Sounds good. Sorry I didn't know `pipeline` module is for simple helper functions. Maybe rename it as something like`utility`?

Reply:
Good point, we could do that. Will change the name after merging this PR then